include/xtensor/xslice.hpp

```diff
@@ -762,6 +762,13 @@ namespace xt
         return slice;
     }
 
+    template <class E, class T>
+    inline auto get_slice_implementation(E& e, xdrop_slice<T>&& slice, std::size_t index)
```

Comment:
@JohanMabille I think this one was forgotten.

Reply:
indeed, good catch!